.changeset/healthy-books-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+add experimental support for `build.split`
```

Comment:
because it does not include all the tasks from the [tasklist](https://github.com/withastro/astro/pull/7464#issue-1771787799), and I want to improve the implementation in the future :)

Reply:
I am not very familiar with experimental flags inside adapters, but if we were to add one, we should pass an explicit property, like:

```js
// astro.config.mjs
import cloudflare from "@astrojs/cloudflare"
export default defineConfig({
	output: "server",
	adapter: cloudflare({
		mode: 'directory',
		experimentalSplit: true
	})
})
```

This is the pattern that we have in core, so we could extend it in adapters too.

Reply:
ok maybe we should not say it is `experimental`, because it actually does not have a current working solution.

_(changeset discussion below)_